backend/app/services/chat/ai_trigger.py

```
@@ -124,15 +124,12 @@ async def _trigger_direct_chat(
     """
     from app.api.ws.events import ServerEvents
 
-    # Emit chat:start event
+    # Emit chat:start event using global emitter for cross-worker broadcasting
     logger.info(f"[ai_trigger] Emitting chat:start event")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)